mods/mobs/dionaea/mob/_nymph.dm

```diff
@@ -85,6 +85,3 @@
 	var/datum/extension/hattable/hattable = get_extension(src, /datum/extension/hattable)
 	if(hattable?.hat)
 		to_chat(user, SPAN_NOTICE("It is wearing [html_icon(hattable.hat)] \a [hattable.hat]."))
-
-/mob/living/carbon/alien/diona/has_dexterity()
```

Comment:
Without this, won't it still have `DEXTERITY_BASE`, since `carbon/alien` doesn't support species?